cloudbuild.yaml

```
@@ -239,12 +247,16 @@ images:
   - 'gcr.io/$PROJECT_ID/java:8-debug'
   - 'gcr.io/$PROJECT_ID/java:11'
   - 'gcr.io/$PROJECT_ID/java:11-debug'
+  - 'gcr.io/$PROJECT_ID/java-debian9:base'
+  - 'gcr.io/$PROJECT_ID/java-debian9:base-debug'
```

```suggestion
  - 'gcr.io/$PROJECT_ID/java:base'
  - 'gcr.io/$PROJECT_ID/java:base-debug'
  - 'gcr.io/$PROJECT_ID/java-debian9:base'
  - 'gcr.io/$PROJECT_ID/java-debian9:base-debug'
```
Reorder these if you want.